pom.xml

```diff
@@ -35,7 +35,7 @@
         <module>specification/amqp_1_0.jms</module>
```

Comment:
Your pom.xml has the wrong value for relativePath, that is why the builds are failing on Travis. relativePath needs to be ../../../pom.xml (one level higher because this project is one level lower than the other projects under specifications).